deb/publish/gen.rb

```diff
@@ -31,6 +31,20 @@
 <pre style="padding:0.5em; margin:1em; background-color:black; color:white">
 deb #{url} binary/
 </pre>
+</p>
+
+<p>
+You will need to explicitly install a Java runtime environment, given the current state of Java 9 is the safest way to
```

Comment:
🐛  I think you mean to say "because Java 9 does not work with Jenkins, this is the safest way" -- this sounds like Java 9 IS supported.